samcli/commands/validate/validate.py

```diff
@@ -57,20 +57,43 @@ class SamTemplate:
     "Create a cfnlintrc config file to specify additional parameters. "
     "For more information, see: https://github.com/aws-cloudformation/cfn-lint",
 )
+@click.option(
+    "--serverless-rules",
+    is_flag=True,
+    help="[DEPRECATED] Enable Serverless Rules for linting validation. "
+    "Requires the cfn-lint-serverless package to be installed. "
+    "Use --extra-lint-rules=\"cfn_lint_serverless.rules\" instead. "
+    "For more information, see: https://github.com/awslabs/serverless-rules",
+)
+@click.option(
+    "--extra-lint-rules",
+    help="Specify additional lint rules to be used with cfn-lint. "
+         "Format: module.path (e.g. 'cfn_lint_serverless.rules')",
+    default=None
+)
```

Comment:
I think you need to pass `multiple=True` to allow the parameter to be passed multiple times (https://click.palletsprojects.com/en/stable/options/#multiple-options and it will be stored as a list).

You mentioned in the documentation that that is actually supported, did you test passing it multiple times? (I imagine the last one will just replace any other one if we don't add the `multiple=True`)